Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'sass-rails'
 gem 'patternfly-sass', "~>1.3.1"
 gem 'bootstrap-datepicker-rails'
 
+# Needed for the Self-Service UX
+gem 'sass'
```

Comment:
@dclarizio @h-kataria @martinpovolny Any issues with including `sass` for the self-service UI?